webportal/webapp/src/assets/js/loginsync.js

```diff
@@ -0,0 +1,4 @@
+function togglePassword(e1, e2) {
```

Comment:
Question: I noticed this webapp doesn't have a lot of Javascript. Wondering if that's a pattern or a coincidence.